website/api/controllers/articles/view-articles.js

```diff
@@ -49,26 +49,64 @@ module.exports = {
         }
       });
     }
-    // Set a currentSection variable for the website header based on how the articles category page is linked to in the header navigation dropdown menus.
-    let currentSection;
-    if(category === 'success-stories'){
-      // If the category is success-stories, highlight the "Platform" dropdown.
-      currentSection = 'platform';
-    } else if(_.contains(['deploy','guides','releases'], category)) {
-      // If the category is deploy, guides, or release, highlight the "Documentation" dropdown.
-      currentSection = 'documentation';
-    } else {
-      // If the category is anything else, highlight the "Community" dropdown.
-      currentSection = 'community';
+
+    let pageTitleForMeta = 'Fleet blog | Fleet for osquery';
+    let pageDescriptionForMeta = 'Read the latest articles written by Fleet';
+    // Create a currentSection variable, this will be used to highlight the header dropdown that this article category lives under.
+    // There are three possible values for this (docuementation, community, and platform), so we'll default to the one with the most article categories (community), and set the value to another section if needed.
+    // If the category is deploy, guides, or releases, currentSection will be set to 'documentation', and if the category is 'success-stories', currentSection will be set to 'platform'.
+    let currentSection = 'community';
+
+    // Set a pageTitleForMeta, pageDescriptionForMeta, and currentSection variable based on the article category.
+    switch(category) {
+      case 'success-stories':
+        pageTitleForMeta = 'Success stories | Fleet for osquery';
+        pageDescriptionForMeta = 'Read about how others are using Fleet and osquery.';
+        currentSection = 'platform';
+        break;
+      case 'deploy':
+        pageTitleForMeta = 'Deployment guides | Fleet for osquery';
+        pageDescriptionForMeta = 'Learn more about how to deploy Fleet.';
```

Comment:
Can you elaborate on this to emphasize that Fleet can be deployed on many infrastructures? Sorry, I'm unsure of the correct terminology, but something like, "Learn how to deploy Fleet on a variety of production environments."